components/navbar/navbar.css

```diff
@@ -137,3 +137,7 @@
 .ampstart-sidebar .ampstart-navbar-trigger {
   line-height: inherit;
 }
+
+.ampstart-headerbar .ampstart-navbar-trigger svg {
```

Comment:
Just `.ampstart-navbar-trigger svg` should be fine. is this because the svg does not bubble up click events?